### ISSUE TITLE
CJ4 keyboard throttle control

### ISSUE THREAD

**Issue by 39crauds33** [Working-Title-MSFS-Mods/fspackages#1295]
I do not have throttle hardware, but have always used F11 key to small decrease and F12 key to small increase throttles.  These keys are assigned this way in "controls" for throttle 1 and throttle 2 as well as throttle alone.  Until SU5 pressing these keys worked both throttles simultaneously and always kept them in perfect sync.  Now these keys randomly move throttles out of sync and sometimes pressing the F11 key to decrease throttles (in CRU mode for example) will accelerate one throttle and decelerate the other and these actions are unpredictable.  Also before SU5 the adjustment could be finely controlled.  Now a single push of the F12 to raise speed slightly (as in descent) jumps the throttle up to TKOF mode!  Is this something that your mod can fix or is there another way to set up the keyboard throttle control.  Any suggestions.  It makes this beautiful mod very difficult to control smoothly without constantly having to correct the throttle positions with my mouse.

**Comment by dga711** [comment]
It seems to me like that is changed sim behavior unfortunately. But I can take a look if there is somethign we can do on the mod end.

**Comment by 39crauds33** [comment]
I did some trial and error and it seems I now have it sorted.  I had set up
separate keys for each engine so I could practice engine outs with
multi-engines.  I then also assigned two single keys to regulate all
throttles at once.  ASOBO has changed the way keyboard responds with the
SU5.  There now are different settings available.  So I made some
reasignments and now I have dual engine control synced again.  So thank you
for your response, but this issue has been resolved.
